onnxruntime/core/framework/allocator.cc

```diff
@@ -43,6 +43,7 @@ bool IAllocator::CalcMemSizeForArrayWithAlignment(size_t nmemb, size_t size, siz
 void* AllocatorDefaultAlloc(size_t size) {
   const size_t alignment = MlasGetPreferredBufferAlignment();
   if (size <= 0) return nullptr;
+  size += MLAS_SYMM_QGEMM_BUF_OVERRUN;
```

Comment:
What if the user uses customized allocator? I think we need to handle it in the computation of buffer size of tensor.